eleven-group/pickup
# Feature/19

Handle image upload in api. 

How to test :

- POST request on http://api.localhost/api/media_objects
- Header Authorization: Bearer YOUR_JWT
- Body form-data
- Key: file, type: file and as value a local file

Expected response : 
```
{
    "data": {
        "id": "/api/media_objects/5",
        "type": "MediaObject",
        "attributes": {
            "contentUrl": "/media/slide-5e82fc7f722dd514979754.png"
        }
    }
}
```
```
curl --location --request POST 'http://api.localhost/api/media_objects' \
--header 'Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE1ODU1OTYyMTMsImV4cCI6MTU4NTY4MjYxMywicm9sZXMiOlsiUk9MRV9BRE1JTiJdLCJ1c2VybmFtZSI6ImFkbWluIn0.UC7tJwvYX4xS2pqfc1irinoFxAHCnZ3IJv_xeO1V-e9u556byVT4HbRiRrB9DcyKdajvoOCrL-Rfn1iMsnwTo3ahTkTjXHbio4VsK18aSja39IM8M22yd8v737PPow3FP1ILEbwnmYefwxq0n34kQiIfPeFujneAbZKEGzmeLLZ637yesTdNap-0WcR1cvByU03gHiSY4P10owqCwKvxXotwSoGLii033b0UmwCKbXDcTagdWUCNvAZlRfjHfWI-rLPSG6lrSqv4a8kNhv-c9aX9T7rqpAmuloFryfngPcHajq8vq4YwNXIhm0hQ9lMPG_gj7e8pA_k4HtpuRT15zZ890A3UILszVaeSWtfWEAaLqTuYuWUkbJwI5CdK-u8Rol7rab5gnHouAYS4-kuAnrmAt_3NY3-XZqlusrhiStxRfSnTQum061b3YvcEkPJCp5zbqAKpUymPfskG-KdvMMIwIvffIy8TDh44swgcDxFZxFPEPo2TxMzywhaHFw1hRAOeA_9Rg4wg3oyiW9p-so3U-zlPdxq9TbWJ_uGKgB1ubzwB4t1DJZT6Xtg1zrUrpm4G6ZL9Vvd1GEDNrj2W7b-jNwGuRI3SOQz7Vq5WF23htaUwnYeirJlaI1mXIWp6G065kzAFLDsmWo9NUirJ1ryTzuIrIoF9vLLifkoAdik' \
--header 'Content-Type: multipart/form-data; boundary=--------------------------430726383068361203875769' \
--form 'file=@/Users/valentinvarveropoulos/Downloads/Slide.png'
```
